classes/class-wc-connect-shipping-label.php

```diff
@@ -164,7 +164,7 @@ public function meta_box( $post ) {
 			wp_enqueue_style( 'wc_connect_admin' );
 
 			?>
-			<div class="wc-connect-admin-container" id="<?php echo esc_attr( $root_view ) ?>"><span class="form-troubles" style="opacity: 0"><?php printf( __( 'Settings not loading? Visit the WooCommerce Connect <a href="%s">debug page</a> to get some troubleshooting steps.', 'woocommerce' ), $debug_page_uri ); ?></span></div>
```

Comment:
Minor: You got rid of `some` here, but not in the shipping-method file. I'd probably get rid of it there too to keep it consistent.